frontend/packages/dev-console/src/components/import/advanced/BuildConfigSection.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { CheckboxField, EnvironmentField } from '../../formik-fields';
+import { CheckboxField, EnvironmentField } from '@console/shared/src/components/formik-fields';
```

Comment:
Recommend we change all of these to be `@console/shared` and export all the formik fields out of the `console-shared/src/components/index.ts` file, following the other styles of what are done there. I don't think there is any need to know the path to where the formik fields are located when it's in a common shared package.

There are quite a few of the path updates, so I won't spam comments :)

Reply:
Done. I've re-exported the components, utils and types from the index and updated the imports.